packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.test.js

```diff
@@ -14,119 +14,160 @@ import {TooltipPortalAttributeName} from "../util/constants.js";
 import TooltipPortalMounter from "./tooltip-portal-mounter.js";
 import TooltipPopper from "./tooltip-popper.js";
 
-import type {Placement} from "../util/types.js";
-
 type TestHarnessProps = {|
-    refChild: (Element | ?React.Component<*, *>) => mixed,
-    placement: Placement,
+    /**
+     * Identifier to give the bubble
+     */
+    bubbleId?: ?string,
```

Comment:
Do we need both `null` and `undefined`?

Reply:
Removed this flexibility. No longer needed.